### PR TITLE
[flex_counter]: Check if plugin lists are empty

### PR DESCRIPTION
**Pull request by marian-pritsak**
Check plugin lists along with counter IDs lists
before trying to remove FC instance

### CODE REVIEW

File: syncd/syncd_flex_counter.cpp [hunk 1]
```diff
@@ -341,7 +341,11 @@ bool FlexCounter::isCounterMapsEmpty()
 {
     SWSS_LOG_ENTER();
 
-    return m_queueCounterIdsMap.empty() && m_portCounterIdsMap.empty() && m_queueAttrIdsMap.empty();
+    return m_queueCounterIdsMap.empty() &&
+           m_portCounterIdsMap.empty() &&
+           m_queueAttrIdsMap.empty() &&
+           m_queuePlugins.empty() &&
+           m_portPlugins.empty();
```
Comment (lguohan) [inline syncd/syncd_flex_counter.cpp]: the function name does not quite match now. also if you look line 326, line 309 in this file, it checks m_queuePlugins/m_portPlugins, are those duplicated?

**Comment by lguohan** [comment]
thanks marian, can you also fix the comments above the isEmpty() function?

**Comment by marian-pritsak** [comment]
Yes, missed them. Done.

File: syncd/syncd_flex_counter.cpp (hunk 1 above)
Comment (marian-pritsak) [reply syncd/syncd_flex_counter.cpp]: Done

**Comment by lguohan** [comment]
@sihuihan88 , can you approved?

**Pull request merged**